portfolio/src/main/java/com/google/sps/data/CalendarGroup.java

```diff
@@ -0,0 +1,126 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/** This class models a group of calendar events. */
+public class CalendarGroup {
+  public List<CalendarEvent> events;
+  public List<TimeRange> allFreeTimeRanges;
+
+  // These two variables represent the start and end of
+  // possible schedule time. When these times are passed in
+  // to instantiate a CalendarGroup class, these times are
```

Comment:
CalendarGroup => CalendarEventsGroup. Or you can just say "instantiate this class"

Reply:
Updated.